ios/AddCalendarEvent.m

```diff
@@ -272,6 +273,33 @@ - (EKEvent *)createNewEventInstance {
     if (options[_allDay]) {
         event.allDay = [RCTConvert BOOL:options[_allDay]];
     }
+    if (options[_alert]) {
+        NSDate *originalDate =  [RCTConvert NSDate:options[_startDate]];
+
+        if ([[RCTConvert NSString:options[_alert]] caseInsensitiveCompare:@"0"] == NSOrderedSame) 
+        { 
+            EKAlarm * alarm = [EKAlarm alarmWithAbsoluteDate:originalDate];
+            event.alarms = @[alarm];
+        }
+        if ([[RCTConvert NSString:options[_alert]] caseInsensitiveCompare:@"1"] == NSOrderedSame) 
+        { 
+            NSDate *alertReminder = [originalDate dateByAddingTimeInterval:-60*5]; 
+            EKAlarm * alarm = [EKAlarm alarmWithAbsoluteDate:alertReminder];
+            event.alarms = @[alarm];
+        }
+        if ([[RCTConvert NSString:options[_alert]] caseInsensitiveCompare:@"2"] == NSOrderedSame) 
+        {
+            NSDate *alertReminder = [originalDate dateByAddingTimeInterval:-60*30]; 
+            EKAlarm * alarm = [EKAlarm alarmWithAbsoluteDate:alertReminder];
+            event.alarms = @[alarm];
+        }
+        if ([[RCTConvert NSString:options[_alert]] caseInsensitiveCompare:@"3"] == NSOrderedSame) 
+        { 
+            NSDate *alertReminder = [originalDate dateByAddingTimeInterval:-60*60]; 
+            EKAlarm * alarm = [EKAlarm alarmWithAbsoluteDate:alertReminder];
+            event.alarms = @[alarm];
```

Comment:
take a good look at the code: there is a lot of duplicates - the ifs look more or less the same, with just small differences. It would be better to remove the duplications and perhaps extract this functionality into a separate function :)